app/views/polls/questions/show.html.erb

```diff
@@ -0,0 +1,92 @@
+<% provide :title do %><%= @question.title %><% end %>
```

Comment:
This view is obsolete. We do not have a question show now.
Please remove this whole file.